### PR TITLE
Weather homepage

### PR DESCRIPTION
**Pull request by R4Ajeti**
Create a weather forecast site using the weather API 'https://openweathermap.org' from the previous lesson. You should be able to search for a specific location and toggle displaying the data in Fahrenheit or Celsius.

### CODE REVIEW

**Review by shloch — CHANGES_REQUESTED** [review]
# hello @R4Ajeti 

I can see you are already a react user. This project is well executed. Good use of asynchronous js

Screen Shot 2020-01-27 at 12 50 15 AM

Great job 

## [required changes]

- which unit of measurement is on display on screen, is it degree CELCIUS or FAHREINHEIT ?? (specify)

- there should be a toggle button to see results in either CELCIUS or FAHREINHEIT ( [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png)) 

- Configure Stickler into this repo ...Since you are using react, [use this](https://github.com/microverseinc/linters-config/tree/master/react-redux)
- Please update your README explaining how to  to have project settup locally 😊

## Waiting for another re-review,

Happy coding

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (bellom) [inline .eslintrc]: You should remove this file so far you're not using it, you're using `eslintrc.json` in `stickler.yml`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @R4Ajeti  :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

**I quote from the previous reviewer:**
```
there should be a toggle button to see results in either CELCIUS or FAHRENHEIT :

Configure Stickler into this repo ...Since you are using react, use this 

Please update your README explaining how to have the project set up locally 😊 ✅ 
```

**Kindly work on the following;**
1. There is a bug using the toggle button.
Clicking the toggle button changes the value of the original result on second search.
Note: First search is working fine, but once I search for another city and click the toggle button it will change the result back to the first search value.

2. You need to fix the linter's error.
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

**Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....**

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by bellom** [comment]
@R4Ajeti kindly reply your messages on slack

Have you tried the commands I sent?
`npx eslint --fix`

**Comment by bellom** [comment]
Have you tried the commands I sent?
`npx eslint --fix`

**Review by bellom — APPROVED** [review]
## Hi @R4Ajeti  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**